assets/apps/customizer-controls/src/builder/scss/_typography-font-pair-section.scss

```diff
@@ -0,0 +1,81 @@
+.customize-control-neve_font_pairings_control {
+  display: list-item !important;
+
+  .neve-font-pair-selector {
+	display: flex;
+	flex-direction: column;
+	padding: 12px 6px;
+
+	.description {
+		padding: 0 0 24px;
+		margin-bottom: 12px;
+		border-bottom: 1px solid #DDD;
+	}
+
+	button {
+		border: 1px solid transparent;
+		border-radius: 4px;
+		padding: 4px 8px;
+		margin-bottom: 12px;
+		display: flex;
+		flex-direction: column;
+		height: 90px;
+		justify-content: space-evenly;
+		color: var(--nv-text-color);
+		background: var(--nv-site-bg);
+
+		&:hover, &focus {
+			border: 1px solid #333;
+			background: var(--nv-dark-bg);
+			color: var(--nv-text-dark-bg);
+			cursor: pointer;
+		}
+
+		&.active {
+		  position: relative;
+		  border: 2px solid var(--wp-admin-theme-color);
+		  &::before {
+			content: "\f147";
+			position: absolute;
+			right: 0;
+			bottom: 0;
+			display: flex;
+			justify-content: center;
+			align-items: center;
+			width: 27px;
+			height: 27px;
+			background-color: var(--wp-admin-theme-color);
+			color: #fff;
+			border-radius: 6px 0 4px 0;
```

Comment:
I think this can be 6px 0 0; to avoid the small visual issue seen in the screenshot attached to the initial PR body: 
![image](https://user-images.githubusercontent.com/15010186/211591228-abd21ad2-b80c-4b98-836e-623af580ccb8.png)